examples/ex41p.cpp

```diff
@@ -0,0 +1,741 @@
+//                                MFEM Example 41 - Parallel Version
+//
+// Compile with: make ex41p
+//
+// Sample runs: mpirun -np 4 ex41p
```

Comment:
Similar to the serial case, we need (at least) two spaces before `mpirun`.

Also, the indentation can be reduced by starting the sample runs on a new line like this:
```c++
// Sample runs:
//    mpirun -np 4 ex41p
//    ...
```